jenkins/ci.opensuse.org/openstack-cleanvm.yaml

```diff
@@ -38,8 +39,8 @@
            ( openstack_project=="Newton" && (image=="SLE_12_SP2" ))
         || ( openstack_project=="Ocata" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
         || ( openstack_project=="Pike" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
-        || ( openstack_project=="Queens" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
-        || ( openstack_project=="Master" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
+        || ( openstack_project=="Queens" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3" || image=="openSUSE-Leap-15.0"))
```

Comment:
I'd prefer if we keep non-master at one sle and max one opensuse distro to test

Reply:
@dirkmueller then we do not catch Queens+Leap15.0 problems (while we still can fix things in Leap15.0). And only having Leap15.0 (which is not released yet) is imo not an option.